src/main/scala/firrtl_interpreter/Driver.scala

```diff
@@ -2,124 +2,44 @@
-    blackBoxFactories: Seq[BlackBoxFactory] = Seq.empty,
-    maxExecutionDepth: Long                 = ExpressionExecutionStack.defaultMaxExecutionDepth,
-    showFirrtlAtLoad:  Boolean              = false,
-    lowCompileAtLoad:  Boolean              = true)
-  extends firrtl.ComposableOptions {
+case class InterpreterTesterCreated(tester: InterpretiveTester) extends DriverExecutionResult
+case class InterpreterTesterFailed(message: String) extends DriverExecutionResult
 
-  def vcdOutputFileName(optionsManager: ExecutionOptionsManager): String = {
-    if(writeVCD) {
-      s"${optionsManager.getBuildFileName("vcd")}"
-    }
-    else {
-      ""
-    }
-  }
-}
-
-trait HasInterpreterOptions {
-  self: ExecutionOptionsManager =>
-
-  var interpreterOptions = InterpreterOptions()
-
-  parser.note("firrtl-interpreter-options")
-
-  parser.opt[Unit]("fint-write-vcd")
-    .abbr("fiwv")
-    .foreach { _ =>
-      interpreterOptions = interpreterOptions.copy(writeVCD = true)
-    }
-    .text("writes vcd execution log, filename will be based on top circuit name")
-
-  parser.opt[Unit]("fint-vcd-show-underscored-vars")
-    .abbr("fivsuv")
-    .foreach { _ =>
-      interpreterOptions = interpreterOptions.copy(vcdShowUnderscored = true)
-    }
-    .text("vcd output by default does not show var that start with underscore, this overrides that")
-
-  parser.opt[Unit]("fint-verbose")
-    .abbr("fiv")
-    .foreach {_ =>
-      interpreterOptions = interpreterOptions.copy(setVerbose = true)
-    }
-    .text("makes interpreter very verbose")
-
-  parser.opt[Unit]("fint-ordered-exec")
-    .abbr("fioe")
-    .foreach { _ =>
-      interpreterOptions = interpreterOptions.copy(setOrderedExec = true)
-    }
-    .text("operates on dependencies optimally, can increase overhead, makes verbose mode easier to read")
 
-  parser.opt[Unit]("fr-allow-cycles")
-    .abbr("fiac")
-    .foreach { _ =>
-      interpreterOptions = interpreterOptions.copy(allowCycles = true)
-    }
-    .text(s"allow combinational loops to be processed, though unreliable, default is ${interpreterOptions.allowCycles}")
-
-  parser.opt[Long]("fint-random-seed")
-    .abbr("firs")
-      .valueName("<long-value>")
-    .foreach { x =>
-      interpreterOptions = interpreterOptions.copy(randomSeed = x)
-    }
-    .text("seed used for random numbers generated for tests and poison values, default is current time in ms")
-
-  parser.opt[Long]("fint-max-execution-depth")
-    .abbr("fimed")
-    .valueName("<long-value>")
-    .foreach { x =>
-      interpreterOptions = interpreterOptions.copy(maxExecutionDepth = x)
-    }
-    .text("depth of stack used to evaluate expressions")
-
-  parser.opt[Unit]("show-firrtl-at-load")
-    .abbr("fisfas")
-    .foreach { _ =>
-      interpreterOptions = interpreterOptions.copy(showFirrtlAtLoad = true)
-    }
-    .text("compiled low firrtl at firrtl load time")
+object Driver {
```

Comment:
Any motivation for not having this extend `firrtl.options.Driver`?

Reply:
Nope, just a miss.